mysql-test/suite/rocksdb/t/drop_table-master.opt

```diff
@@ -1,4 +1,3 @@
 --rocksdb_max_subcompactions=1
---rocksdb_info_log_level=info_level
```

Comment:
@georgelorchpercona  yes. this is the same issue (and therefore likely not a problem due to your port).  We have an issue internally to track this, but not yet externally (feel free to open and assign to me if you'd like).  For now if you need the test to pass you can also remove the manual compaction in the test, as I believe it's not necessary for the test correctness.